custom/src/main/java/co/elastic/otel/config/DynamicConfiguration.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package co.elastic.otel.config;
+
+import co.elastic.otel.ElasticLogRecordExporter;
+import co.elastic.otel.ElasticMetricExporter;
+import co.elastic.otel.ElasticSpanExporter;
+
+public class DynamicConfiguration {
+  private static DynamicConfiguration INSTANCE = new DynamicConfiguration();
```

Comment:
For reviewer, this class is essentially what the OpAMP client would use, or extend or be implemented into